ui/apps/dashboard/src/app/(dashboard)/settings/team/UserCreator.tsx

```diff
@@ -67,7 +67,14 @@ export function UserCreator() {
 
     const res = await createUser({ input });
     if (res.error) {
-      toast.error('Failed to create user');
+      toast.error('Failed to create user', {
+        description: res.error.graphQLErrors?.[0]?.message,
+        action: {
+          label: 'Contact Support',
```

Comment:
Does this create a button? I don't see it in the screenshot

Reply:
Yep, I didn't update the screenshot:

![Arc 2023-11-10 at 15 55 22@2x](https://github.com/inngest/inngest/assets/4291707/6e9d5a03-ddc8-441b-be12-65d2fd036c8c)